cmd/scollector/collectors/aws.go

```diff
@@ -0,0 +1,98 @@
+package collectors
+
+import (
+	"fmt"
+	"time"
+
+	"bosun.org/metadata"
+	"bosun.org/opentsdb"
+	"bosun.org/slog"
+
+	"github.com/awslabs/aws-sdk-go/aws"
+	"github.com/awslabs/aws-sdk-go/gen/ec2"
+	//"github.com/awslabs/aws-sdk-go/gen/elb"
```

Comment:
WIP, but as a heads up before commit you need to remove all commented out code. Also using goimports gets rid of the pain of manually removing and adding imports when you need them